src/pages/EditMonitor.vue

```diff
@@ -930,6 +931,9 @@ message HealthCheckResponse {
   </soap:Body>
 </soap:Envelope>` ]);
             }
+            if (this.monitor && this.monitor.httpBodyEncoding && this.monitor.httpBodyEncoding === "form") {
+                return this.$t("Example:", [ `key1=value1&key2=value2` ]);
```

Comment:
The linter does like it this way 😉 

```suggestion
                return this.$t("Example:", [ "key1=value1&key2=value2" ]);
```